src/tokens/TokenDetails.tsx

```diff
@@ -136,7 +136,21 @@ function PriceInfo({ token }: { token: TokenBalance }) {
   )
 }
 
-function Actions({ token }: { token: TokenBalance }) {
+export const onPressCicoAction = (token: TokenBalance, flow: CICOFlow) => {
+  const tokenSymbol = token.symbol
+  // this should always be true given that we only show Add / Withdraw if a
+  // token is CiCoCurrency, but adding it here to ensure type safety
+  if (isCicoToken(tokenSymbol)) {
```

Comment:
there's an open question on where Withdraw should navigate right?

Reply:
This is still an open question @nityarao responded in [Figma](https://www.figma.com/file/OE6RFzqXAMqU9ygHtNYyof/Assets-Page-(Multichain)?node-id=2490%3A11056&mode=dev#579471570).

Reply:
I left a comment on the thread. This can be fixed in a follow up PR